app/models/spree_avatax_certified/response.rb

```diff
@@ -0,0 +1,32 @@
+module SpreeAvataxCertified
+  class Response
+    def initialize(response)
+      @response = response
+    end
+
+    def success?
+      return false if error?
+      @response['ResultCode'] == 'Success' || @response[:result_code] == 'Success'
+    end
+
+    def error?
+      @response == 'error in Tax' || @response['ResultCode'] == 'Error' || @response[:result_code] == 'Error'
+    end
+
+    def total_tax
+      @response['TotalTax'] || @response[:total_tax]
+    end
+
+    def tax_lines
+      @response['TaxLines'] || @response[:tax_lines][:tax_line]
```

Comment:
Why would you have different data at this point? 

Better to standardize it in advance, and not to check either or in every method.